python/foglamp/services/core/api/scheduler.py

```diff
@@ -27,9 +27,11 @@
     
     | GET POST        | /foglamp/schedule                                         |
     | GET PUT DELETE  | /foglamp/schedule/{schedule_id}                           |
+    | GET             | /foglamp/schedule/enable/{schedule_id}                    |
```

Comment:
PUT?

Reply:
Agreed, this should not be a GET. Also the URL would be better as /foglamp/schedule/{schedule_id}/enable